model/sharing/open_note.go

```diff
@@ -79,10 +79,6 @@ func (o *Opener) GetResult(memberIndex int, readOnly bool) (jsonapi.Object, erro
 }
 
 func (o *Opener) openLocalNote(memberIndex int, readOnly bool) (*apiNoteURL, error) {
-	// If the note came from another cozy via a sharing that is now revoked, we
-	// may need to recreate the trigger.
-	_ = setupTrigger(o.Inst, o.File.ID())
```

Comment:
I think we should keep this line for existing notes from a revoked sharing without a trigger.

Reply:
Sounds like a good idea indeed.